ci/builders/README.md

```diff
@@ -154,231 +143,237 @@ An archive component is used to tell the recipes which artifacts are generated b
-                   "flutter/build/archives:embedder",
-               ]
-           },
+    "config": "host_debug",
+    "targets": [
+        "flutter/build/archives:artifacts",
+        "flutter/build/archives:embedder",
+    ]
+},
 ```
 
-In the example above the ninja command will use the configuration for host\_debug and will build artifacts and embedder targets as described by the [flutter/build/archives/BUILD.gn](https://cs.opensource.google/flutter/engine/+/master:build/archives/BUILD.gn) file.
-
-##### Tests
-
-This section of the build configuration will be referred to as the local tests. This section contains a list of dictionaries with configurations for the scripts and parameters used to run tests inside the current build unit. These tests should not reference or use anything outside of the commit checkout or the outputs generated by running the gn and ninja sections of the build config.
-
-```
-          "tests": [
-            {
-                "language": "python3",
-                "name": "Host Tests for host_debug_impeller_vulkan",
-                "parameters": [
-                    "--variant",
-                    "host_debug_impeller_vulkan",
-                    "--type",
-                    "impeller-vulkan",
-                    "--engine-capture-core-dump"
-                ],
-                "script": "flutter/testing/run_tests.py",
-                "type": "local"
-            }
-        ]
+In the example above the ninja command will use the configuration for
+host\_debug and will build artifacts and embedder targets as described
+by the
+[flutter/build/archives/BUILD.gn](https://cs.opensource.google/flutter/engine/+/master:build/archives/BUILD.gn)
+file.
+
+#### Tests
+
+This section of the build configuration is also known as local tests. It
+contains a list of dictionaries with configurations for scripts and
+parameters used to run tests inside the current build unit. These tests
+should not reference or use anything outside of the commit checkout or
+the outputs generated by running gn and ninja sections of the build
+configuration.
+
+```yaml
+"tests": [
+   {
+       "language": "python3",
+       "name": "Host Tests for host_debug_impeller_vulkan",
+       "parameters": [
+           "--variant",
+           "host_debug_impeller_vulkan",
+           "--type",
+           "impeller-vulkan",
+           "--engine-capture-core-dump"
+       ],
+       "script": "flutter/testing/run_tests.py"
+   }
+]
 ```
 
 Description of the fields:
 
-* **Language,** the executable used to run the script, e.g. python3.
-* **Name,** the name of the step running the script.
-* **Parameters**, flags or parameters passed to the script.
-* **Script**, the path to the script to execute relative to the checkout directory.
-* **Type**, the test type. (Deprecate? Test on the build config will be always local)
+* **language** - the executable used to run the script, e.g. python3, bash.
```

Comment:
What are the allowed values here other than `python3` and `bash`?

Reply:
Added more info, any executables in the path can be used as language. An example is using `"language" : "cat"` and `"script": "/a/b/c.txt"` will be translated to `cat /a/b/c.txt`.

Reply:
Would this mean that it is the executable rather than the language?